index.js

```diff
@@ -7,17 +7,22 @@ const bplistParser = require('bplist-parser');
 const untildify = require('untildify');
 
 const bplist = pify(bplistParser);
-const settings = path.join(userHome, '/Library/Preferences/com.runningwithcrayons.Alfred-Preferences-3.plist');
+const settingsV3 = path.join(userHome, '/Library/Preferences/com.runningwithcrayons.Alfred-Preferences-3.plist');
+const settingsLatest = path.join(userHome, '/Library/Preferences/com.runningwithcrayons.Alfred-Preferences.plist');
 
 module.exports = async () => {
-	const exists = await pathExists(settings);
+	const latestExists = await pathExists(settingsLatest);
+	const v3Exists = await pathExists(settingsV3);
 
-	if (!exists) {
-		throw new Error(`Alfred preferences not found at location ${settings}`);
+	if (!(latestExists || v3Exists)) {
```

Comment:
IMO it's easier to read if you did it like this

```suggestion
	if (!latestExists && !v3Exists) {
```